### DISCUSSION TITLE
how to refer a person or invite someone

### DISCUSSION THREAD

**Discussion by Sivayogeith** [codewars/codewars.com#3142] [Q&A] [answered]
I tried search the site, its docs and google but i cant find how to invite someone else over to codewars (for referral honor)

**Comment by Madjosz** [comment, marked as answer]
Click on "Learn more" in "Earn extra honor..." panel on the dashboard and it will reveal your invitation link.
![Referral program](https://github.com/codewars/codewars.com/assets/28844868/00502ba9-68b1-4b73-a10c-25949af21328)
But please note that there is currently bug #2923

**Reply by Sivayogeith** [reply]
Thanks! I thought it might bring me to a docs page. Sorry 😅
